src/utils/wallet.ts

```diff
@@ -228,6 +228,57 @@ const wallet = {
       addrMap[address.toString()] = index;
     }
     return addrMap;
+  },
+
+  /**
+   * Get Hathor address at specific index
+   *
+   * @param {string} xpubkey The xpubkey. We expect the xpub in the last derivation path (change).
```

Comment:
Suggestion: "The xpubkey in the last derivation path (change level according to BIP0044)".